### PR TITLE
forgotPassword bug fix 

### PR DESCRIPTION
**Pull request by minsunny**
https://nanopay.atlassian.net/browse/NP-4599

nanopay: https://github.com/nanoPayinc/NANOPAY/pull/12843

1. fixed reset password email not being sent out 
2. fixed a bug that reset password link in reset password email gets auto log-in 
3. fixed errorPageview not working with error message, updated to use theme logo and ui consistency like SuccessPasswordView
4. added "isTokenValid()" in token service to see if token is expired, used, not existing etc.

2&3&4 updates in requestLogin() controller.js

5. tableWidth in token data not to be cut off

Screen Shot 2021-06-03 at 11 37 06 PM

Screen Shot 2021-06-03 at 11 38 22 PM

Screen Shot 2021-06-03 at 11 39 34 PM

Screen Shot 2021-06-07 at 1 56 28 PM

Screen Shot 2021-06-07 at 1 56 10 PM

Screen Shot 2021-06-07 at 1 55 56 PM

Screen Shot 2021-06-07 at 1 55 30 PM

### CODE REVIEW

**Review by smirnovaka — APPROVED** [review]
lgtm

File: src/foam/util/Emails/EmailsUtility.java [hunk 1]
```diff
@@ -55,6 +55,7 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
     AppConfig appConfig = (AppConfig) x.get("appConfig");
     if ( user != null ) {
       userX = x.put("subject", new Subject.Builder(x).setUser(user).build());
+      userX = x.put("group", user.findGroup(x));
```
Comment (jlhughes) [inline src/foam/util/Emails/EmailsUtility.java]: This and the next two lines, can we just lookup group once?
Group group = user.findGroup(x),  then use it for getAppConfig and and the userX setup?

Comment (jlhughes) [inline src/foam/util/Emails/EmailsUtility.java]:
```suggestion
      userX = userX.put("group", user.findGroup(userX));
```

Comment (minsunny) [reply src/foam/util/Emails/EmailsUtility.java]: Updated

Comment (minsunny) [reply src/foam/util/Emails/EmailsUtility.java]: updated

**Review by jlhughes — APPROVED** [review]

**Pull request merged**